lib/zendesk_apps_support/package.rb

```diff
@@ -278,6 +279,30 @@ def app_js
       read_file('app.js')
     end
 
+    def location_icons
+      {}.tap do |location_icons|
+        manifest.locations.each do |host, locations_for_host|
+          next unless [Product::SUPPORT.name].include?(host)
+
+          locations_for_host.keys.each do |location|
+            next unless ['top_bar', 'nav_bar'].include?(location)
+            location_icons[host] ||= {}
+            location_icons[host][location] = if (has_file?("assets/icon_#{location}.svg"))
+              { 'svg' => "icon_#{location}.svg" }
+            elsif (has_file?("assets/icon_#{location}_inactive.png"))
+              {
+                'inactive' => "icon_#{location}_inactive.png",
```

Comment:
Like above, worth extracting a constant for `ICON_STATES` and iterating over that rather than the duplication?